docker/Dockerfile.compile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && apt-get install -y \
+  cmake make \
+  clang g++ \
+  libboost-iostreams-dev libboost-system-dev \
+  libgmp-dev \
+  liblua5.2-dev libluajit-5.1-dev \
```

Comment:
I dont know if its just me or luajit is completely ignore with setup like this. Cmake uses lua 5.2 only anyway I think